pages/welcome/welcome.py

```diff
@@ -0,0 +1,162 @@
+from dash import dcc, html
+import dash
+import dash_bootstrap_components as dbc
+import plotly.express as px
+
+# register the page
+dash.register_page(__name__, path="/welcome", order=5)
+
+# This is just a stand-in until we wire up a more interesting visualization.
+example_fig = px.scatter(
+    x=[1, 2, 3, 4, 5, 6],
+    y=[1, 2, 3, 4, 5, 6],
+    color=["blue", "red", "green", "yellow", "purple", "brown"],
+    labels={"x": "X Values", "y": "Y Values"},
+    size_max=19,
+    size=[1, 1, 1, 1, 1, 1],
+)
+
+example_fig.update_layout(
+    title={
+        "text": "Example Figure",
+        "y": 0.9,
+        "x": 0.5,
+        "xanchor": "center",
+        "yanchor": "top",
+    }
+)
+
+layout = dbc.Container(
+    className="welcome_container",
+    children=[
+        html.Div(
+            className="welcome_message welcome_section",
+            children=[
+                # html.H1("Welcome to 8Knot"),
+                html.Img(src="assets/logo-color.png"),
+                html.P(
+                    """
+                    Open source communities are difficult to understand. 8Knot serves community stakeholders by
+                    providing a platform to host advanced analysis of community behavior through higher-order metrics and visualizations.
+                    """
+                ),
+            ],
+        ),
+        html.Div(
+            className="welcome_content_section shadow",
+            children=[
+                html.Div(
+                    className="welcome_section_header",
+                    children=[html.P("Pages Available")],
+                ),
+                html.Div(
+                    className="welcome_section_content",
+                    children=[
+                        html.Div(
+                            className="pages_overview_container shadow",
+                            children=[
+                                html.H2("Overview"),
+                                html.P(
+                                    """
+                                    Track large community trends over time based on common contributions.
```

Comment:
Ah, I wrote a comment but it was 'Pending' because it was in 'Review' mode.

Since these are full sentences, they get punctuation. Here's a reference site that has similar sub-sections:
https://www.sigstore.dev/